.docker/volumes/mysql/create-schema.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+SCHEMA_DIR=/tmp/schema
+
+mysql_exec=( mysql -uroot -proot )
```

Comment:
Are we using `root` as the MySQL root password here?
I guess root password can be changed by setting a value to `MYSQL_ROOT_PASSWORD` ENV variable.
Shall we use `${MYSQL_ROOT_PASSWORD}` as the root password?

Reply:
Damn! Nice catch!